composer.json

```diff
@@ -14,19 +14,18 @@
   "support": {
     "email": "info@simpleshop.cz"
   },
-  "autoload": {
-    "psr-4": {
-      "SimpleShop\\Api\\": "libs/"
-    }
-  },
   "require": {
     "php": ">=7.3.0",
     "ext-curl": "*",
     "ext-json": "*",
-    "composer/ca-bundle": "^1.2",
     "vyfakturuj/vyfakturuj-api-php": ">=2.5.1"
   },
   "suggest": {
     "composer/ca-bundle": "Umožňuje ověřit SSL certifikáty serveru"
+  },
```

Comment:
Přečetl jsem, že `"suggest"` je rekomendace k rozšíření, ale není mi jasné, jak ho uživateli knihovny použijí? Mají tento balíček dodat ručně, nebo k tomu existuje nějaký composer příkaz, jako `composer install --suggested`?

Reply:
Hm, Composer 2 tuto funkci dost potlačil. Dříve to vedlo k tomu, že po instalaci tohoto balíčku se v konzoli zobrazil přehled doporučení pro tento balíček. Nic víc, žádné automatické instalace zde nejsou. Služí to pouze pro "dobrovolnou" závislost, která není vynucována, viz: https://getcomposer.org/doc/03-cli.md#suggests

Nyní se zobrazuje po instalaci jen nenápadná informace, kterou je potřeba "rozbalit" pomocí příkazu `composer suggests`.

![](https://cdn.jakub-boucek.cz/screenshot/210610-i00cb.png)

Reply:
Děkuju za odpověď